lib/Mojolicious/Command/openapi.pm

```diff
@@ -68,6 +69,7 @@ sub run {
     }
   );
 
+  warn "[".__PACKAGE__."] Calling '".ref($self->_client)."::$op' ($content)\n" if DEBUG;
```

Comment:
I don't see how this helps, since it should be implicit which method it calls. Also, `$content` can be `undef, so that line might result in uninitialized warnings.